language-php.md

```diff
@@ -2,12 +2,77 @@
 
 We should follow best practices accepted by the community.
 As mentioned in _[php the right way][1]_, PHP-FIG, the PHP Framework Interop Group, announced PSR.
+Some of the style guides are inspired by [CodeIgniter Framework](https://www.codeigniter.com/user_guide/general/styleguide.html)
 
 We decided to follow [PSR-2].
 
 [1]: http://www.phptherightway.com/
 [PSR-2]: https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md
 
+## Table of Contents
+
+1. [SQL Queries](#sql-queries)
+2. [Localized Texts](#localized-texts)
+3. [Debug Codes](#debug-codes)
+4. [Code Sizes](#code-sizes)
+5. [Operators](#operators)
+
+## SQL Queries
+
+Although were using an ORM, there are some cases that have to do some complex sql statements and these can be done using raw MYSQL query.
```

Comment:
? Although we’re using an [ORM](https://en.wikipedia.org/wiki/Object-relational_mapping), there are cases where complex SQL statements need raw MySQL queries.